src/components2/MapLegend/MapLegend.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { breakpoints } from '~/styles/utils';
+
+import LegendExpanded from './LegendExpanded';
+
+interface Props {
```

Comment:
My rule of thumb is to always use `type` (and nearly never interface). The "why" was too complex to remember, I just follow it :-).

Reply:
I'll fix these issues in the according PR : #960